components/02-molecules/banner/grand-hero/_yds-grand-hero.scss

```diff
@@ -30,13 +30,15 @@ $break-grand-hero-banner: tokens.$break-m;
   }
 }
 
-.grand-hero-banner__image {
+.grand-hero-banner__image,
```

Comment:
I'm surprised adding a new class here with a comma didnt cause linting issues with specificity in the nested elements. At any rate, the combination of these class with video,img will result in styles that never apply. Example: `.grand-hero-banner__image video {}` - I would probably change these wrapping classes to `.grand-hero-banner__media` with a video or img modifier if you need to separate these styles. Not a big deal but thought I would mention this other approach to simplify these styles.

Reply:
@callinmullaney I just went ahead and separated them. :+1: